osbs/build/build_request.py

```diff
@@ -95,6 +98,9 @@ def set_params(self, **kwargs):
 
         self.base_image = kwargs.get('base_image')
         self.low_priority_node_selector = kwargs.get('low_priority_node_selector')
+        if kwargs.get('platform'):
+            self.platform = kwargs.get('platform')
```

Comment:
You could squash this commit in if you like: https://github.com/twaugh/osbs-client/commit/d2f33aea6c0d73bdfb8adf645101643b40c09106